libs/application/template-api-modules/src/lib/modules/templates/document-provider-onboarding/document-provider-onboarding.service.ts

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I would like to avoid the use of `any` where we can.

Until we take the next step and refactor the state machine out of templates and into the template-api-modules we can do something like:

```typescript
interface Contact {
  name?: string
  email: string
  phoneNumber: string
}

// ...
const applicant = get(application.answers, 'applicant') as string
const adminContact = get(
  application.answers,
  'administrativeContact',
) as Contact
const techContact = get(application.answers, 'technicalContact') as Contact
const helpdesk = get(application.answers, 'helpDesk') as Contact
```

Reply:
had to cast to `unknown` first